src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String[] STATISTICS_FIELDS = {"supply", "buy", "result"};
+    private static final int SUPPLY_POSITION = 0;
+    private static final int BUY_POSITION = 1;
+    private static final int RESULT_POSITION = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = readFromFile(fromFileName);
+        int[] statisticsValues = calculateStatistics(fileContent);
+        writeToFile(statisticsValues, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
Please avoid using 1-2 letters variable names. And names that based on class. Name should describe what data is inside:
```suggestion
        StringBuilder fileContent = new StringBuilder();
```